src/backend/feed/processor.js

```diff
@@ -102,7 +103,10 @@ function articlesToPosts(articles, feed) {
   return Promise.all(
     articles.map(async article => {
       try {
-        await Post.createFromArticle(article, feed);
+        const id = await Post.createFromArticle(article, feed);
+        // Use the returned id to get the 'text' from the post and index it
+        const { text } = await Post.byId(id);
```

Comment:
Can you move this indexing step into `createFromArticle()` here: https://github.com/Seneca-CDOT/telescope/blob/master/src/backend/data/post.js#L134?  We could use `Promise.all()` there and let redis and elastic both work in parallel.